util/topgen/version_file.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
```

Comment:
I think this is missing deleting `version_file.py` in the root util folder.

Reply:
Oh yes definitely, I think the right thing to do is to create an actual module (under `util` dir) out of `version_file.py` so that it can be referenced both from `reggen` and `topgen`. I think these modules are the only ones relying on version_file.

Reply:
I've updated the PR, which makes it even simpler, since I moved `version_file.py` to its own module. That allows every Python module and Python script to directly invoke `import version_file` or `from version_file import ...`.